qa/TL1_custom_cpp_lib/test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+test_body() {
+    mkdir -p /tmp/custom_cpp_lib
+    git clone https://github.com/NVIDIA/DALI /tmp/dali
+    pushd /tmp/custom_cpp_lib
+    echo "
+#include <dali/c_api.h>
+int main() {
```

Comment:
I think you can move this to separate file. There is no strong need to put it in this test.sh.

Reply:
I figured it would be easier to create custom file in temporary location this way. Unless it's easier to do some trick that I'm not aware of?